infra/aws/govcloud.md

```diff
@@ -0,0 +1,199 @@
+    "AccountName": "spacecats-govcloud-id",
+    "State": "IN_PROGRESS",
+    "RequestedTimestamp": 1582040671.363
+  }
+}
+```
+
+You can then run the following command to get the account ID of your new
+GovCloud account:
+
+```console
+$ aws organizations describe-create-account-status \
+  --create-account-request-id car-8d813980526511ea7184124ac536023e
+```
+
+You'll get output that looks like this:
+
+```json
+{
+  "CreateAccountStatus": {
+    "Id": "car-8d813980526511ea7184124ac536023e",
+    "AccountName": "spacecats-govcloud-id",
+    "State": "SUCCEEDED",
+    "RequestedTimestamp": 1582040671.446,
+    "CompletedTimestamp": 1582040736.794,
+    "AccountId": "123456789000",
+    "GovCloudAccountId": "000987654321"
+  }
+}
+```
+
+In that output, `AccountId` is the account number for the dummy account,
+and the `GovCloudAccountId` is the account number for your new GovCloud
+account.
+
+Now that you have that, you can assume the `OrganizationAccountAccessRole`
+in the new account from an account with sufficient privileges in the
+*GovCloud* organization master account. *Note that this is the case even
+though at this point the GovCloud organization does not show up if you
+look at the organization membership in the console.*
+
+### Adding Your New GovCloud Account to Your GovCloud Organization
+
+To add the account you just created to your GovCloud organization, run
+this command with your appropriate credentials from the GovCloud
+organization's master account.
+
+```console
+$ aws organizations invite-account-to-organization \
+  --target Id=000987654321,Type=ACCOUNT
+```
+
+This will return output that looks like this:
+
+```text
+{
+  "Handshake": {
+    "Id": "h-d6ac6154ef6a42e68513d0d8460af49a",
+
+...
+```
+
+Using the credentials in the new account, run this command:
+
+```console
+$ aws organizations accept-handshake \
+  --handshake-id h-d6ac6154ef6a42e68513d0d8460af49a
+```
+
+The account will now be in your GovCloud organization. You will want to
+import this account into your Terraform state for the GovCloud
+organization master account like so:
+
+```hcl
+resource "aws_organizations_account" "spacecats_govcloud_id" {
+  name      = "spacecats-govcloud-id"
+  email     = "spacecats-infra+spacecats-govcloud-id@example.com"
+  parent_id = aws_organizations_organizational_unit.spacecats_govcloud.id
+  tags = {
+    Automation = "Terraform"
+  }
+}
+```
+
+Then run this command:
+
+```console
+$ terraform import aws_organizations_account.spacecats_govcloud_id 000987654321
+```
+
+### Handling GovCloud Dummy Accounts in AWS Commercial
+
+As stated above, when you use the AWS CLI to create your new GovCloud
+accounts, it will also create a dummy account with the same account name
+in your commercial AWS organization. In order to prevent misuse of these
+accounts (and any confusion, since they have the same account alias as
+your GovCloud account), we can lock these down with the `suspended` OU
```

Comment:
NICE

Reply:
Yeah, I was pretty happy when I figured out this was fine.